content/docs/contributing/to-docs/style-guide/format.md

```diff
@@ -195,48 +123,36 @@ Syntax highlighting in the code block should conform to the [contrast guidelines
 
 When writing code blocks, make sure to include a language identifier. For C++ code blocks, use the `cpp` identifier.
 
-### Mark placeholders with angle brackets
-
-Use angle brackets for placeholders. Use the text within the brackets to tell the reader what a placeholder represents. For example:
-
-```shell
-git push origin <your-branch-name>
-```
 
-### Bold user interface elements
+### Links
 
-Do | Don't
-:--| :-----
-Choose **Edit**. | Click "Edit".
-.. the **Play** button. | ... the Play button.
+#### Relative and absolute links
 
-### Bold input combinations
+Write links without the file extension *.md*.
```

Comment:
```suggestion
Write links without the file extension `.md`.
```

Minor formatting nit for the formatting page; Since these are directions regarding a filename I'd suggest they be code-formatted rather than bolded.

Reply:
This is right, thanks for catching! Would be ironic to not follow the style guide in the style guide itself.